source/collections/Vector2D.ooc

```diff
@@ -35,19 +36,74 @@ Vector2D: class <T> {
 	free: override func {
 		gc_free(this _backend)
 	}
-	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
+	_elementPosition: func (row, column: Int, columnCount := this columnCount) -> Int {
 		result := columnCount * row + column
```

Comment:
It seems to be working fine:
```
lang_Numbers__Int Vector2D__Vector2D__elementPosition(Vector2D__Vector2D* this, lang_Numbers__Int row, lang_Numbers__Int column, lang_Numbers__Int columnCount) {
    
    {
        lang_Numbers__Int result = columnCount * row + column;
        return result;
    }
}
```
But I will change the line to just `columnCount * row + column`since the assignment to a vairable seems to be redundant?